apps/participants/tests/test_views.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+import pytest
+from django.http import QueryDict
+from django.urls import reverse
+
+from apps.experiments.models import ParticipantData
+from apps.utils.factories.experiment import ExperimentSessionFactory, ParticipantFactory
+
+
+@pytest.mark.django_db()
+def test_edit_participant_data(client, team_with_users):
+    participant = ParticipantFactory(team=team_with_users)
+    team = participant.team
+    session = ExperimentSessionFactory(participant=participant, team=team, experiment__team=team)
+    user = participant.team.members.first()
+    data = {"name": "A"}
+    participant_data = ParticipantData.objects.create(
+        team=team, content_object=session.experiment, participant=participant, data=data
+    )
+    client.login(username=user.username, password="password")
+
+    url = reverse(
+        "participants:edit-participant-data",
+        kwargs={
+            "team_slug": participant.team.slug,
+            "participant_id": participant.id,
+            "experiment_id": session.experiment.id,
+        },
+    )
+
+    data = QueryDict("", mutable=True)
```

Comment:
this shadows the `data` var on line 17 which makes like 34 confusing: ` data.update({"data": json.dumps(data)})`